File: sdk/java/src/main/java/com/hedera/hashgraph/sdk/AccountBalanceResponse.java
package com.hedera.hashgraph.sdk;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

// TODO: Hbar
// TODO: tokens
@JsonIgnoreProperties("$type")
public final class AccountBalanceResponse {
    @JsonProperty
    public AccountId accountId;

    @JsonProperty
    public long balance;
}

File: sdk/kotlin/src/main/kotlin/com/hedera/hashgraph/sdk/AccountBalanceResponse.kt
package com.hedera.hashgraph.sdk

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

// TODO: Hbar
// TODO: tokens
@JsonIgnoreProperties("\$type")
class AccountBalanceResponse constructor(
    @JsonProperty("accountId")
    @JvmField
    val accountId: AccountId,

    @JsonProperty("balance")
    @JvmField
    val balance: Long
)
